src/CONST/index.ts

```diff
@@ -1822,7 +1822,7 @@ const CONST = {
     ATTACHMENT_THUMBNAIL_WIDTH_ATTRIBUTE: 'data-expensify-width',
     ATTACHMENT_THUMBNAIL_HEIGHT_ATTRIBUTE: 'data-expensify-height',
     ATTACHMENT_DURATION_ATTRIBUTE: 'data-expensify-duration',
-
+    ATTACHMENT_IMAGE_DEFAULT_NAME: 'shared_image.png',
```

Comment:
why do we need this btw? What scenario does the `validatedFile?.uri` not exist?

Reply:
If an error occurs during validation, or if we are not in a case where validation should be performed, this field will be empty. That’s why we need to guard against such cases.